src/main/java/org/bonitasoft/console/server/listener/StudioWatchdogListener.java

```diff
@@ -131,10 +128,38 @@ private static boolean isAlive(final int port, final long timer) throws IOExcept
 
             }
         }
-        if (LOGGER.isLoggable(Level.FINE)) {
-            LOGGER.log(Level.FINE, "Bonita Studio JVM is alive");
-        }
+
+        LOGGER.debug("Bonita Studio JVM is alive");
         return true;
     }
 
+    private static void shutdownTomcat() {
+        try {
+            // We expect to have only one MBean server. Get it.
+            MBeanServer mBeanServer = MBeanServerFactory.findMBeanServer(null).get(0);
+
+            // Get Server object name
+            ObjectName name = new ObjectName("Catalina", "type", "Server");
+
+            // Get the server object
+            Server server = (Server) mBeanServer.getAttribute(name, "managedResource");
+
+            // Get adress and port on which we can send the shutdown command
+            String address = server.getAddress();
+            int port = server.getPort();
+
+            // Get the shutdown command that we need to send
+            String shutdown = server.getShutdown();
+
+            // Connect and send the shutdown command    	    
+            Socket clientSocket = new Socket(address, port);
+            clientSocket.getOutputStream().write(shutdown.getBytes());
+            clientSocket.getOutputStream().flush();
+            clientSocket.getOutputStream().close();
+            clientSocket.close();
+        } catch (Exception e) {
+            LOGGER.error("Error while trying to shutdown Tomcat from watchdog web application", e);

```

Comment:
Shouldn't we call System.exit(-1); in catch clause in order to keep the old behavior in case of issue?